src/server/api/routes/users/v2/follow.js

```diff
@@ -43,45 +41,67 @@ router.get('/followees', authenticateV2Login, async (req, res, next) => {
   }
 });
 
-router.post(
-  '/followees',
-  authenticateV2Login,
-  checkEmailHasVerified,
-  async (req, res, next) => {
-    try {
-      const { user } = req.session;
-      const { creator } = req.query;
-      if (!isValidFollowee(user, creator)) {
-        res.status(400).send('INVALID_CREATOR_ADDRESS');
-        return;
-      }
-      await db.runTransaction(async t => {
-        const userRef = walletUserCollection.doc(user);
-        const userDoc = await t.get(userRef);
-        const { email, emailUnconfirmed } = userDoc.data();
-        if (!email) {
-          if (emailUnconfirmed) {
-            throw new Error('EMAIL_UNCONFIRMED');
-          } else {
-            throw new Error('EMAIL_NOT_SET_YET');
-          }
-        }
-        await t.update(userRef, {
-          followees: FieldValue.arrayUnion(creator),
-        });
-      });
-      publisher.publish(PUBSUB_TOPIC_MISC, req, {
-        logType: 'UserCreatorFollow',
-        type: 'wallet',
-        user,
-        creatorWallet: creator,
+router.get('/interacted', authenticateV2Login, async (req, res, next) => {
+  try {
+    const { user } = req.session;
+    const userDoc = await walletUserCollection.doc(user).get();
+    if (!userDoc.exists) {
+      res.json({ followees: [] });
```

Comment:
Should be same as below?
```suggestion
      res.json({ interactedCreators: [] });
```